packages/compass-e2e-tests/tests/logging.test.js

```diff
@@ -9,7 +9,8 @@ function cleanLog(log) {
       '$1'
     )
   );
-  for (const entry of log) {
+  for (let i = 0; i < log.length; i++) {
+    const entry = log[i];
     expect(entry.t.$date).to.be.a('string');
     delete entry.t; // Timestamps vary between each execution
```

Comment:
Unrelated, but it looks like this function modifies the passed in log - was that intentional? I changed the behavior in https://github.com/mongodb-js/compass/pull/2438 but wasn't sure if it was for a reason I missed.

Reply:
It’s taking a copy of the log right at the beginning anyway, so yeah, this was intentional… we can see whichever PR gets merged first and then adjust the other